package/boot/uboot-rockchip/Makefile

```diff
@@ -25,22 +25,32 @@ define U-Boot/rockpro64-rk3399
   NAME:=RockPro64
   BUILD_DEVICES:= \
     pine64_rockpro64
+  BL31=$(STAGING_DIR_IMAGE)/rk3399_bl31.elf
   DEPENDS:=+PACKAGE_u-boot-rockpro64-rk3399:arm-trusted-firmware-rockchip
   PKG_BUILD_DEPENDS:=arm-trusted-firmware-rockchip
 endef
 
+define U-Boot/rock64-rk3328
+  BUILD_SUBTARGET:=armv8
+  NAME:=Rock64
+  BUILD_DEVICES:= \
+    pine64_rock64
+  BL31=$(STAGING_DIR_IMAGE)/rk3328_bl31.elf
+  DEPENDS:=+PACKAGE_u-boot-rock64-rk3328:arm-trusted-firmware-rockchip
```

Comment:
Why is this conditional dependency needed here? 
Isn't this sufficent:
`
DEPENDS:=+arm-trusted-firmware-rockchip
`